<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentTrip</class>
 <widget class="QWidget" name="CurrentTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCommandLinkButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Destination">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>256</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ArrivedBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrived</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EndTripBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>End Trip</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>79</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Passenger :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Cost :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Distance :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Status  :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>71</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="Passenger">
      <property name="text">
       <string>Pass</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Cost">
      <property name="text">
       <string>Cost</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Distance">
      <property name="text">
       <string>Dis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Status">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-4</y>
     <width>441</width>
     <height>451</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>c4.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>BackBtn</zorder>
  <zorder>Destination</zorder>
  <zorder>label_5</zorder>
  <zorder>ArrivedBtn</zorder>
  <zorder>EndTripBtn</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
